docs/modules/ROOT/partials/cmos-health-checks.adoc

```diff
@@ -248,4 +248,18 @@ If the problem persists, then please contact Couchbase Technical Support.
 
 *Further Reading*: xref:7.0@server:metrics-reference:data-service-metrics.adoc#kv_ep_storage_age_second[Data Service Metrics], xref:7.0@server:install:sizing-general.adoc#sizing-data-service-nodes[Sizing Data Service Nodes]
 
+[#CB90071]
+=== Compaction Failure (CB90071)
+
+*Background*: Auto-Compaction can be set to run either at a specific time, or when bucket storage becomes too low.
```

Comment:
I didn't know that was a thing, and I can't find a setting for it - here's the auto-compaction settings on a 7.0.3 cluster, not sure which of these is about being low on disk space (though may well be missing something obvious so please correct me if I'm wrong):
![image](https://user-images.githubusercontent.com/2904440/153431958-f838ce03-167e-43d2-a434-2e7e6849ea02.png)
![image](https://user-images.githubusercontent.com/2904440/153432014-4c417b1e-41f2-462f-a339-33b025828ee9.png)

Reply:
Ah my bad, I mixed up fragmentation levels with storage amount. I'll change it where appropriate to mention fragmentation levels instead of storage space. Good find, thank you!